src/app/components/ATIAnalytics/params/index.ts

```diff
@@ -167,32 +160,21 @@ export const buildATIEventTrackingParams = ({
   data,
   atiData,
 }: ATIConfigurationDetailsProviders) => {
-  try {
-    const { pageType } = requestContext;
-    if (atiData && isMigrated(pageType)) {
-      return buildPageATIParams({
-        atiData,
-        requestContext,
-        serviceContext,
-      });
-    }
-
-    const buildParams = createBuilderFactory(
+  const { pageType } = requestContext;
+  if (atiData && isMigrated(pageType)) {
+    return buildPageATIParams({
+      atiData,
       requestContext,
-      pageTypeParamBuilders,
-    );
-
-    return buildParams(data as PageData, requestContext, serviceContext);
-  } catch (error: unknown) {
-    const { message } = error as Error;
+      serviceContext,
+    });
+  }
 
-    // eslint-disable-next-line no-console
-    console.error(
-      `ATI Event Tracking Error: Could not parse tracking values from page data:\n${message}`,
-    );
+  const buildParams = createBuilderFactory(
+    requestContext,
+    pageTypeParamBuilders,
+  );
 
-    return {};
```

Comment:
Hmm maybe I should leave in the error handling just in case something occurs within `buildPageATIParams` or `buildParams`